docs/sites.yml

```diff
@@ -8323,6 +8323,31 @@
   built_by: Jason Di Benedetto
   built_by_url: https://jason.dibenedetto.fyi
   featured: false
+- title: Yoseph.tech
+  main_url: https://www.yoseph.tech
+  url: https://www.yoseph.tech/compilers
+  source_url: https://github.com/radding/yoseph.tech_gatsby
+  description: >
+    Yoseph.tech is a personal blog centered around technology and software engineering
+  categories:
+    - Technology
+    - Web Development
+    - Open Source
+  built_by: Yoseph Radding
+  built_by_url: https://github.com/radding
+  featured: false
+- title: Gildform
+  main_url: https://www.gildform.com
+  url: https://www.gildform.com
+  source_url:
+  description: >
+    Gildform is an on demand jewelry manaufacturer, taking 3D models uploaded through the site and converting it to a real piece of jewelry
+  categories:
+    - eCommerce
+    - Manufacturing
```

Comment:
We don't have `Manufactoring` in our categories whitelist - see for list of available ones https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml#L64-L118
```suggestion
```